forge-gui/res/cardsfolder/upcoming/dust_animus.txt

```diff
@@ -0,0 +1,12 @@
+Name:Dust Animus
+ManaCost:1 W
+Types:Creature Spirit
+PT:2/3
+K:Flying
+K:ETBReplacement:Other:CounterChoice
+SVar:CounterChoice:DB$ PutCounter | ETB$ True | CounterNum$ 2 | CounterType$ P1P1 | ConditionCheckSVar$ X | ConditionCompare$ GE5 | SubAbility$ LifelinkCounter | SpellDescription$ If you control five or more untapped lands, CARDNAME enters the battlefield with two +1/+1 counters and a lifelink counter on it.
+SVar:LifelinkCounter:DB$ PutCounter | ConditionCheckSVar$ X | ConditionCompare$ GE5 | ETB$ True | CounterType$ Lifelink
+SVar:X:Count$Valid Land.YouCtrl+untapped
```

Comment:
@tool4ever this probably needs LastStateBattlefield?

Reply:
Made the engine smart enough for it sometime ago:
https://github.com/Card-Forge/forge/blob/80b3526ec1885f59e3985d17ded17aa7d910564c/forge-game/src/main/java/forge/game/ability/AbilityUtils.java#L1912

but always good to retest this from time to time